src/app/welcome/page.tsx

```diff
@@ -0,0 +1,104 @@
+"use client"
+
+import { useState } from "react"
+import { useRouter } from "next/navigation"
+import {
+  Carousel,
+  CarouselApi,
+  CarouselContent,
+  CarouselItem,
+} from "@/components/ui/carousel"
+
+export default function Welcome() {
+  const [carouselApi, setCarouselApi] = useState<CarouselApi>()
+  const [currIndex, setCurrIndex] = useState(0)
+  const router = useRouter()
+
+  const onNext = () => {
+    if (!carouselApi) return
+
+    if (carouselApi.canScrollNext()) {
+      carouselApi.scrollNext()
+      setCurrIndex((i) => i + 1)
+    } else {
+      router.push("/dashboard")
+    }
+  }
+
+  const isLastCard = carouselApi && !carouselApi.canScrollNext()
+
+  return (
+    <div className="flex flex-col items-center pt-20 pb-4 bg-backgroundSecondary min-h-lvh">
+      <div className="bg-background w-2/3 p-16 text-center rounded-3xl items-center flex flex-col gap-6">
+        <div className="py-0.5 px-2 bg-backgroundSecondary rounded-full">
+          Welcome
+        </div>
+        <Carousel setApi={setCarouselApi}>
+          <CarouselContent>
+            <WelcomeCard card="intro" />
+            <WelcomeCard card="projects" />
+            <WelcomeCard card="funding" />
+          </CarouselContent>
+        </Carousel>
+        <Dots total={3} current={currIndex} />
+        <button
```

Comment:
annoying nit but the `Button` in the shadcn components/ui dir should be styled for this (can grab similar variant/etc. from my PR)

Reply:
Oh so true bestie